### ISSUE TITLE
It doesnt work on play mode :/

### ISSUE THREAD

**Issue by nidaynere** [sebastianhein/urp-ssao#1]
Unity Version: 2019.3.0b12
URP: 7.1.5

The second one is in playing mode. Issue is SSAOmaterial is null in Execute method in playing mode.

![resim](https://user-images.githubusercontent.com/36076974/80265104-1ba5e380-869f-11ea-94aa-dc633631c773.png)
![resim](https://user-images.githubusercontent.com/36076974/80265180-5576ea00-869f-11ea-9533-a213214eedbd.png)

**Comment by CorvidCoven** [comment]
Yeah i get this issue as well, it works in scene view, but not in play mode, I also have HDR turned off.

**Comment by oscarbrooks** [comment]
+1, also does not appear in Standalone build. I had a play around with changing the Render Pass Event but no luck.
Really nice and easy to set up though, and the results look nice + good options

**Comment by CorvidCoven** [comment]
Doesn't really matter if it only works in the editor hahaha, was digging, playing around with the script and shader, and I can't see where it goes wrong :/

**Comment by oscarbrooks** [comment]
During play mode (assuming the same with the build), I'm seeing that ssaoMaterial is null, causing the Execute method to exit early

https://github.com/sebastianhein/urp-ssao/blob/4958115f46d11e781af478ae78732c55f2bed446/Scripts/SSAO.cs#L71-L73

Going to have a look around as to why that may be

**Comment by oscarbrooks** [comment]
Ok so I have found something
```csharp
#if UNITY_EDITOR
            if (!Application.isPlaying)
            {
                ResourceReloader.TryReloadAllNullIn(this, "Packages/com.sbstn_hn.rendererfeatures.urp-ssao");
            }
#endif
```
If you remove the #if directive `#if UNITY_EDITOR` and the `!Application.isPlaying` check, it should run in the editor paused and play mode and should also run in a build.

I'm not too familiar with custom render passes and SRP in general, so I don't know about the performance implications of this. It seems to be working for now

**Comment by CorvidCoven** [comment]
Ahhhh, i see, i took that entire section out thinking it was specifically saying only show it in editor, kinda dumb shouldve looked at it closer, also use Unity's profiler if you wanna look at how performant it is

Thank you!

**Comment by MBarnulf** [comment]
Hi, I have done that, can confirm it solves it, maybe this should be corrected in the repo ?
